easybuild/easyconfigs/r/RFdiffusion/RFdiffusion-1.1.0_fix-find-packages.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/setup.py b/setup.py
```

Comment:
For a patch, please include a short description of what it does and author information.

This will mean the checksum will need updating in https://github.com/easybuilders/easybuild-easyconfigs/pull/18304/files#diff-a4b869cde7f93bdbee308dd26726ef1701df2c4fef364540bfcc1d6bc3736c44R48-R49